ipatests/test_webui/data_loginscreen.py

```diff
@@ -8,6 +8,9 @@
 PASSWD_ITEST_USER = '12345678'
 PASSWD_ITEST_USER_NEW = '87654321'
 
+ROOT_PKEY = 'root'
+PASSWD_ADMIN = 'Secret123'
```

Comment:
Just a small issue.
As seen in the [`ipa-test-config.yaml`](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/deb70a1e-5ef8-11ed-807b-fa163e04db38/ipa-test-config.yaml) generated for the `ipaserver` topology, the `admin` password is actually `Secret.123`